src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -69,7 +81,7 @@ export class HuiDialogEditCard extends LitElement {
 
 declare global {
   interface HTMLElementTagNameMap {
-    "hui-dialog-edit-card": HuiDialogEditCard;
+    "hui-dialog-edit-config": HuiDialogEditCard;
```

Comment:
@bramkragten my fireEvent work found this bug because it verifies that the dialog tag that you try to register is a known element 👍

Reply:
`edit-card` is better than `edit-config` I think, but good thing you found it.